src/main/java/com/google/sps/model/BudgetResponse.java

```diff
@@ -1,10 +1,25 @@
 package com.google.sps.model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class BudgetResponse {
-    public void addToHTML(double budget, double contribution) {
-        
+    List<String> htmlToIntegrate;
+
+    public BudgetResponse(){
+        htmlToIntegrate = new ArrayList<String>();
+    }
+
+    public String writeHTMLLine(double budget, double contribution){
+        return String.format("<p>Trip Budget: %f, Your Expected Contribution: %f</p>", budget, contribution);
```

Comment:
could you return the two numbers as json? It'd be easier to reuse the parameters elsewhere

Reply:
Good idea, i'll work on this